<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>728</width>
    <height>857</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>728</width>
    <height>857</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="PolygonDisplayView" name="graphicsView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>541</width>
          <height>340</height>
         </size>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOn</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOn</enum>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustToContents</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="label_filePath">
          <property name="text">
           <string>File path : </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_Load">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>24</height>
           </size>
          </property>
          <property name="text">
           <string>Load</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_Refresh">
          <property name="text">
           <string>Refresh</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_Slider">
            <property name="minimumSize">
             <size>
              <width>70</width>
              <height>21</height>
             </size>
            </property>
            <property name="text">
             <string>     Scale</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="doubleSpinBox_scale">
            <property name="minimumSize">
             <size>
              <width>67</width>
              <height>25</height>
             </size>
            </property>
            <property name="minimum">
             <double>0.010000000000000</double>
            </property>
            <property name="maximum">
             <double>1000.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
            <property name="value">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_showDataMarkers">
          <property name="minimumSize">
           <size>
            <width>151</width>
            <height>22</height>
           </size>
          </property>
          <property name="text">
           <string>Show Data Markers</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextEdit" name="textEdit_filePath">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>549</width>
          <height>67</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>67</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QCheckBox" name="checkBox_Decompose">
          <property name="minimumSize">
           <size>
            <width>170</width>
            <height>31</height>
           </size>
          </property>
          <property name="text">
           <string>Decompose polygon</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>448</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QCheckBox" name="checkBox_slope">
          <property name="minimumSize">
           <size>
            <width>160</width>
            <height>31</height>
           </size>
          </property>
          <property name="text">
           <string>Line slope tol</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBox_slopeTol">
          <property name="minimumSize">
           <size>
            <width>83</width>
            <height>31</height>
           </size>
          </property>
          <property name="decimals">
           <number>4</number>
          </property>
          <property name="minimum">
           <double>0.000100000000000</double>
          </property>
          <property name="maximum">
           <double>1000.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.001000000000000</double>
          </property>
          <property name="value">
           <double>0.001000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>78</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_arcTol">
          <property name="minimumSize">
           <size>
            <width>160</width>
            <height>22</height>
           </size>
          </property>
          <property name="text">
           <string>Arc radius tol</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBox_curvatureTol">
          <property name="minimumSize">
           <size>
            <width>83</width>
            <height>31</height>
           </size>
          </property>
          <property name="decimals">
           <number>4</number>
          </property>
          <property name="minimum">
           <double>0.000100000000000</double>
          </property>
          <property name="maximum">
           <double>1000.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.001000000000000</double>
          </property>
          <property name="value">
           <double>0.010000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QCheckBox" name="checkBox_splineTol">
          <property name="minimumSize">
           <size>
            <width>170</width>
            <height>31</height>
           </size>
          </property>
          <property name="text">
           <string>Spline tolerance</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBox_SplineTol">
          <property name="minimumSize">
           <size>
            <width>83</width>
            <height>31</height>
           </size>
          </property>
          <property name="decimals">
           <number>4</number>
          </property>
          <property name="minimum">
           <double>0.000100000000000</double>
          </property>
          <property name="maximum">
           <double>1000.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.001000000000000</double>
          </property>
          <property name="value">
           <double>0.050000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>78</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_sharpAngle">
          <property name="minimumSize">
           <size>
            <width>160</width>
            <height>31</height>
           </size>
          </property>
          <property name="text">
           <string>Sharp angle tol (degrees)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBox_angleTol">
          <property name="minimumSize">
           <size>
            <width>83</width>
            <height>31</height>
           </size>
          </property>
          <property name="decimals">
           <number>4</number>
          </property>
          <property name="minimum">
           <double>0.000100000000000</double>
          </property>
          <property name="maximum">
           <double>180.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
          <property name="value">
           <double>10.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QCheckBox" name="checkBox_firstOrder">
          <property name="minimumSize">
           <size>
            <width>170</width>
            <height>31</height>
           </size>
          </property>
          <property name="text">
           <string>Tolerance 1 (not used)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBox_tolFirstOrder">
          <property name="minimumSize">
           <size>
            <width>83</width>
            <height>31</height>
           </size>
          </property>
          <property name="decimals">
           <number>4</number>
          </property>
          <property name="minimum">
           <double>0.000100000000000</double>
          </property>
          <property name="maximum">
           <double>1000.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.000100000000000</double>
          </property>
          <property name="value">
           <double>0.000100000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_8">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>78</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_secondOrder">
          <property name="minimumSize">
           <size>
            <width>160</width>
            <height>31</height>
           </size>
          </property>
          <property name="text">
           <string>Tolerance 2 (not used)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBox_tolSecondOrder">
          <property name="minimumSize">
           <size>
            <width>83</width>
            <height>31</height>
           </size>
          </property>
          <property name="decimals">
           <number>4</number>
          </property>
          <property name="minimum">
           <double>0.000100000000000</double>
          </property>
          <property name="maximum">
           <double>1000.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.000100000000000</double>
          </property>
          <property name="value">
           <double>0.000100000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>13</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_convexDecomp">
        <property name="minimumSize">
         <size>
          <width>581</width>
          <height>91</height>
         </size>
        </property>
        <property name="title">
         <string>Convex decomposition (NOT IMPLEMENTED)</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QCheckBox" name="checkBox_LOD">
           <property name="minimumSize">
            <size>
             <width>131</width>
             <height>22</height>
            </size>
           </property>
           <property name="text">
            <string>Level of detail</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="doubleSpinBox_LOD">
           <property name="minimumSize">
            <size>
             <width>83</width>
             <height>31</height>
            </size>
           </property>
           <property name="decimals">
            <number>0</number>
           </property>
           <property name="minimum">
            <double>1.000000000000000</double>
           </property>
           <property name="maximum">
            <double>99.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>249</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox_concavityTol">
           <property name="minimumSize">
            <size>
             <width>111</width>
             <height>22</height>
            </size>
           </property>
           <property name="text">
            <string>Concavity tol</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="doubleSpinBox_concavityTol">
           <property name="minimumSize">
            <size>
             <width>83</width>
             <height>31</height>
            </size>
           </property>
           <property name="decimals">
            <number>4</number>
           </property>
           <property name="maximum">
            <double>1.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.010000000000000</double>
           </property>
           <property name="value">
            <double>0.010000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>338</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_Apply">
          <property name="minimumSize">
           <size>
            <width>101</width>
            <height>31</height>
           </size>
          </property>
          <property name="text">
           <string>Apply</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_OK">
          <property name="minimumSize">
           <size>
            <width>101</width>
            <height>31</height>
           </size>
          </property>
          <property name="text">
           <string>OK</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>728</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionMain">
   <property name="text">
    <string>Main</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>PolygonDisplayView</class>
   <extends>QGraphicsView</extends>
   <header location="global">polygondisplayview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
